Marlin/SanityCheck.h

```diff
@@ -64,11 +64,12 @@
       #error EXTRUDERS must be 1 with HEATERS_PARALLEL.
     #endif
 
-    #ifdef Y_DUAL_STEPPER_DRIVERS
+    // Dual axis only when less then two extruders defined and an second extruder is available
+    #if ((EXTRUDERS>2) || (defined(Y_DUAL_STEPPER_DRIVERS) && (!defined(E2_STEP_PIN))))
```

Comment:
An RUMA board has 6 Stepper drivers. Its possible to define Two extruders and dual Y axis. That's the reason i changed this check. I use the third extruder slot to serve dual Y axis.